src/test/java/nearsoft/academy/bigdata/recommendation/TransformData.java

```diff
@@ -0,0 +1,76 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import java.io.*;
+
+/**
+ * Created by AMDA on 30/03/2017.
+ */
+class TransformData
+{
+    public String filenameOutput;
+
+    //List Manage
+    public ManageList manageList = new ManageList();
+
+    public void setOutputFile(String file){
+
+        String [] temp = file.split("\\.");
+
+        filenameOutput = temp[0] + "Output." + temp[1];
+    }
+
+    public void transformToPreferenceFile(String file)
+    {
+        try{
+            //Reader
+            FileReader fr = new FileReader(file);
+            BufferedReader br = new BufferedReader(fr);
+
+            //Writer
+            FileWriter fw = new FileWriter(filenameOutput);
+            BufferedWriter bw = new BufferedWriter(fw);
+
+            //Line control
+            String line;
+
+            StringBuilder sb = new StringBuilder();
+
+            int index = 0, totalIndexes = 0;
```

Comment:
My mistake. I initialized these variables because is a local variable for the method. I know that only the class variables initialize their own value to default.